### PR TITLE
xserver: qtile as desktopManager

### PR DESCRIPTION
**Pull request by kamilchm**

### CODE REVIEW

File: nixos/modules/services/x11/desktop-managers/qtile.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+with lib;
+
+let
+  cfg = config.services.xserver.desktopManager.qtile;
+in
+
+{
+  options = {
+    services.xserver.desktopManager.qtile = {
+      enable = mkOption {
```
Comment (jagajaga) [inline nixos/modules/services/x11/desktop-managers/qtile.nix]: Please use `mkEnableOption`.

**Comment by lethalman** [comment]
Ok as long as it works for you :)

**Comment by kamilchm** [comment]
It works, but it needs qtile package before ;)
https://github.com/NixOS/nixpkgs/pull/9033

**Comment by edolstra** [comment]
I now get:
```
$ nix-instantiate --dry-run nixos/ -A system
error: attribute ‘qtile’ missing, at /home/eelco/Dev/nixpkgs/nixos/modules/services/x11/desktop-managers/qtile.nix:6:9
```

File: nixos/modules/services/x11/desktop-managers/qtile.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.desktopManager.qtile;
```
Comment (joachifm) [inline nixos/modules/services/x11/desktop-managers/qtile.nix]: This never gets declared, as far as I can tell

**Comment by lethalman** [comment]
Reverted in 203dad10fec6b9285bc063f2aa66fc97c17849c3 . Please make another PR and make sure you test it this time.

**Pull request merged**